stoplight.js

```diff
@@ -0,0 +1,7 @@
+$('#stoplight').find('div').on('click',clickedLight);
+isBulbLit = $(this).attr('class');
+console.log(isBulbLit)
+// function clickedLight(){
+//   previouslyLit.removeClass('bulb active').addClass('bulb');
+//   $(this).addClass('bulb active');
+// }
```

Comment:
You are on the right path here! I think you just had to break down the problem into a couple more steps.